Source/Core/loadWithXhr.js

```diff
@@ -152,12 +152,20 @@ define([
         xhr.onload = function() {
             if (xhr.status === 200) {
                 if (defined(xhr.response)) {
-                    deferred.resolve(xhr.response);
+                    if (responseType === 'json' && typeof xhr.response === 'string') { //IE doesn't support responseType = 'json'
+                        try{
+                            deferred.resolve(JSON.parse(xhr.response));
+                        } catch (e) {
+                            deferred.reject(new RuntimeError('unknown XMLHttpRequest response type.'));
```

Comment:
I think this should just be `deferred.reject(e)` so that whatever the real browser error is comes though.